src/DependencyInjection/AlgoliaSearchExtension.php

```diff
@@ -15,61 +16,73 @@
 /**
  * This is the class that loads and manages your bundle configuration.
  *
- * @link http://symfony.com/doc/current/cookbook/bundles/extension.html
+ * @see http://symfony.com/doc/current/cookbook/bundles/extension.html
  */
-class AlgoliaSearchExtension extends Extension
+final class AlgoliaSearchExtension extends Extension
 {
     /**
-     * {@inheritdoc}
+     * @param array            $configs
+     * @param ContainerBuilder $container
+     *
+     * @throws \Exception
```

Comment:
Would it be possible to be more precise about the exceptions that could be thrown?